CRM/Financial/BAO/FinancialTypeAccount.php

```diff
@@ -157,30 +151,20 @@ public static function del($financialTypeAccountId, $accountId = NULL) {
    * @param int $paymentInstrumentValue
    *   Payment instrument value.
    *
-   * @return array|null|string
+   * @return null|int
    */
-  public static function getInstrumentFinancialAccount($paymentInstrumentValue = NULL) {
```

Comment:
Just a small note -- this was technically a change in the function signature (dropping the `NULL` default). Not a huge deal because it's an internal function, but there was a stale reference in `sql/GenerateData.php` (which is used for `regen.sh`) where it calls:

```php
 $financialAccountId = CRM_Financial_BAO_FinancialTypeAccount::getInstrumentFinancialAccount();
```

To work around it, I grepped for`getInstrumentFinancialAccount` and saw a test written with the hard-coded value `getInstrumentFinancialAccount(4)`. Not sure if it's the right thing, but using `4` does seem to get `regen.sh` running.